cmd/tmsUpload.go

```diff
@@ -110,7 +111,10 @@ func runTmsUpload(config tmsUploadOptions, communicationInstance tms.Communicati
 		return fmt.Errorf("mta file %s not found", mtaPath)
 	}
 
-	description := config.CustomDescription
+	description := tms.DEFAULT_TR_DESCRIPTION
```

Comment:
The config.CustomDescription always contains a value (either custom or git commit id). You can check that setting in tmsUpload.yaml file. I think, I tried that it works, but please also try by yourself.

Reply:
If you call binary without --customDescription description is empty and the upload fails

Reply:
Understood. All fine then.